runtime-manager/modules/ROOT/pages/rtm-agent-allowlists.adoc

```diff
@@ -63,13 +63,19 @@ Ports, IP addresses, and hostnames are required for outbound connectivity. Inbou
 
 == Static IP Addresses
 
+[NOTE]
+As part of our continuous infrastructure improvement, Mulesoft introduces a new set of static IP addresses in Runtime Manager to augment existing ones. To avoid service disruption, allow the IP addresses or domains, depending on your network policy, through your organization's firewall before February 1st, 2024, as listed below.
```

Comment:
```suggestion
As part of our continuous infrastructure improvement, MuleSoft introduces a new set of static IP addresses in Runtime Manager to augment existing ones. To avoid service disruption, allow the IP addresses or domains, depending on your network policy, through your organization's firewall before February 1st, 2024, as listed below.
```